tests/trading/test_claimTradingProceeds.py

```diff
@@ -93,12 +95,14 @@ def test_redeem_shares_in_binary_market(kitchenSinkFixture, universe, cash, mark
     logs = []
     captureFilteredLogs(kitchenSinkFixture.chain.head_state, kitchenSinkFixture.contracts['Augur'], logs)
 
-    # redeem shares with a1
-    initialLongHolderETH = kitchenSinkFixture.chain.head_state.get_balance(tester.a1)
-    claimTradingProceeds.claimTradingProceeds(market.address, sender = tester.k1)
-    # redeem shares with a2
-    initialShortHolderETH = kitchenSinkFixture.chain.head_state.get_balance(tester.a2)
-    claimTradingProceeds.claimTradingProceeds(market.address, sender = tester.k2)
+    with ETHDelta(expectedMarketCreatorFees, tester.a0, kitchenSinkFixture.chain, "Market creator fees not paid"):
```

Comment:
Style: `EthDelta` or `EtherDelta`.  Only first letter of acronym/abbreviation should be capitalized.